README.md

```diff
@@ -554,3 +554,13 @@ It should appear on the map as you refresh your local browser.
 - `ERROR: for X Cannot create container for service X: Invalid bind mount spec "<path>": Invalid volume specification: '<volume spec>'`. If you get this error after running `docker-compose up` on Windows, you should tell `docker-compose` to properly understand Windows paths by setting the environment variable `COMPOSE_CONVERT_WINDOWS_PATHS` to `0` by running `setx COMPOSE_CONVERT_WINDOWS_PATHS 0`. You will also need a recent version of `docker-compose`. We have successfully seen this fix work with [v1.13.0-rc4](https://github.com/docker/toolbox/releases/tag/v1.13.0-rc4). More info here: https://github.com/docker/compose/issues/4274.
 
 - No website found at `http://localhost:8000`: This can happen if you're running Docker in a virtual machine. Find out docker's IP using `docker-machine ip default`, and replace `localhost` by your Docker IP when connecting.
+
+- FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory: You can configure the memory allocation using `NODE_OPTIONS=--max-old-space-size=4096` in the "build-release" script in package.json, example: "export `NODE_OPTIONS=--max-old-space-size=4096` NODE_ENV=production && webpack --bail --mode production". Also update to the latest node version.
+
+- Expected linebreaks to be 'LF' but found 'CRLF': Fixed by adding `"linebreak-style": 0` in "rules" in .estlintrc
+
+- ERROR: for electricitymap-contrib_mockserver_1  Cannot start service mockserver: OCI runtime create failed: container_linux.go:345: starting container process caused "process_linux.go:424: container init caused \"rootfs_linux.go:58: mounting \\\".../server.py\\\" to rootfs \\\"/mnt/sda1/var/lib/docker/overlay2/.../merged\\\" at \\\".../server.py\\\" caused \\\"not a directory\\\"\"": unknown: Are you trying to mount a directory onto a file (or vice-versa)? Check if the specified host path exists and is the expected type
+   - Check that the project is cloned under C:/Users/
```

Comment:
Do you know why this solves the problem? More importantly, `C:/Users` implies user is on Windows - perhaps we can create a "**Windows specific**" subsection in "**Troubleshooting**" and move this bullet point there?

Reply:
I read Unix using 'sudo' commands which is similar to increased admin privileges. My project was cloned outside my C:/Users directory, after I moved it there it worked. Might be a coincidence, but could help others